docs/src/app/components/pages/components/DropDownMenu/README.md

```diff
@@ -0,0 +1,6 @@
+## Drop Down Menu
+
+To find out more about the `DropDownMenu` component please visit the Material Design's
+specifications **WHERE IS IT IN THE SPEC?!**.
```

Comment:
There isn't I don't think, just the second-last animation here: https://www.google.com/design/spec/components/menus.html#menus-usage (Textfield dropdown).